app/src/test/java/com/axehai/weatherscope/domain/use_case/set_active_location/SetActiveLocationUseCaseTest.kt

```
@@ -0,0 +1,60 @@
+
+
+}
+
+class FakeRepository : ActiveLocationRepository {
+
+	var activeLocation: ActiveLocation? = null
+		private set
+
+
+	override fun observeActiveLocation(): Flow<ActiveLocation> {
+		TODO("Not yet implemented")
```

detekt (1.23.8)

[warning] The NotImplementedDeclaration should only be used when a method stub is necessary. This defers the development of the functionality of this function. Hence, the `NotImplementedDeclaration` should only serve as a temporary declaration. Before releasing, this type of declaration should be removed.

(detekt.exceptions.NotImplementedDeclaration)